proto/provenance/oracle/v1/params.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package provenance.oracle.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/provenance-io/provenance/x/oracle/types";
+
+option java_package        = "io.provenance.oracle.v1";
+option java_multiple_files = true;
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+}
```

Comment:
If there aren't any params, I say we just leave it out completely.

The `x/params` module is deprecated anyway, so probably best to not add something new to it.